.github/workflows/test.yml

```diff
@@ -0,0 +1,76 @@
+jobs:
+  build:
+    name: build
+    strategy:
+      matrix:
+        os: [macos-11]
+        python-version: [ 3.8 ]
+    runs-on: ${{ matrix.os }}
+    env:
+      CP_MYSQL_TEST_HOST: "127.0.0.1"
+      CP_MYSQL_TEST_DB: "cellprofiler_test"
+      CP_MYSQL_TEST_USER: "root"
+      CP_MYSQL_TEST_PASSWORD: "None"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          architecture: x64
+          python-version: ${{ matrix.python-version }}
+      - name: Get pip cache dir
+        id: pip-cache-dir
+        run: |
+          echo "::set-output name=dir::$(pip cache dir)"
+      - name: Cache pip
+        uses: actions/cache@v3
+        id: cache-pip
+        with:
+          path: ${{ steps.pip-cache-dir.outputs.dir }}
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/setup.py') }}
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '14.0.1' # The JDK version to make available on the path.
+          java-package: jdk
+          architecture: x64
+      - name: Initial installs
+        run: |
+          pip install pyinstaller
+          pip install --upgrade pip setuptools wheel
+          pip install numpy>=1.20.1
+          git clone https://github.com/CellProfiler/python-javabridge.git ~/python-javabridge
+          git clone https://github.com/CellProfiler/python-bioformats.git ~/python-bioformats
+          git clone https://github.com/CellProfiler/centrosome.git ~/centrosome
+          git clone https://github.com/CellProfiler/core.git ~/core
+          # git clone https://github.com/CellProfiler/CellProfiler.git ~/cellprofiler
+      - name: Linux - install mysql and build tools
+        env:
+          PATH: $PATH:/home/ubuntu/.local/bin
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          sudo apt install -y make gcc build-essential libgtk-3-dev
+          sudo apt-get install -y default-libmysqlclient-dev libnotify-dev libsdl2-dev
+      - name: Mac - setup mysql
+        env:
+          LC_ALL: "en_US.UTF-8"
+        if: startsWith(matrix.os, 'macos')
+        run: |
+          brew install mysql
+          mysql.server start
+          mysql --host=$CP_MYSQL_TEST_HOST --user=$CP_MYSQL_TEST_USER --execute="CREATE DATABASE $CP_MYSQL_TEST_DB;" --skip-password
+          # mysql --host=$CP_MYSQL_TEST_HOST --user=$CP_MYSQL_TEST_USER --execute="CREATE DATABASE $CP_MYSQL_TEST_DB;" --password=$CP_MYSQL_TEST_PASSWORD
```

Comment:
Any reason to keep this?

Reply:
No, at least for now. I had some trouble around [this line](https://github.com/CellProfiler/CellProfiler/blob/master/tests/modules/test_exporttodatabase.py#L114), but I decided it was outside of the scope of this PR, so I'll remove it for now.